src/zzz_od/application/charge_plan/charge_plan_config.py

```
@@ -48,10 +97,116 @@ def __post_init__(self) -> None:
+        if self.target_material_count <= 0:
+            return '目标材料数必须大于 0'
+        return None
+
+    @property
+    def is_finished(self) -> bool:
+        """计划是否已达到所选运行方式的目标。"""
+        if self.is_material_count_plan:
+            invalid_reason = self._material_count_invalid_reason()
+            if invalid_reason is not None:
+                log.warning(
+                    f'材料数量计划配置非法，按已完成处理 '
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)